Userland/Libraries/LibWeb/HTML/History.idl

```diff
@@ -1,5 +1,12 @@
+// https://html.spec.whatwg.org/multipage/history.html#the-history-interface
 [Exposed=Window]
 interface History {
+    readonly attribute unsigned long length;
```

Comment:
This is not mentioned in the commit message, consider splitting this PR up into multiple commits